docs/security_groups.md

```diff
@@ -8,8 +8,10 @@ Kops will still output any differences in the managed and your own Security Grou
 This is convenient for determining policy changes that need to be made when upgrading Kops.
 **Using Managed Security Groups will not output these differences, it is up to the user to track expected changes to policies.**
 
-*NOTE: Currently Kops only supports using existing Security Groups for every instance group and Load Balancer in the Cluster, not a mix of existing and managed Security Groups.
+NOTE: 
+- *Currently Kops only supports using existing Security Groups for every instance group and Load Balancer in the Cluster, not a mix of existing and managed Security Groups.
 This is due to the lifecycle overrides being used to prevent creation of the Security Groups related resources.*
+- *Kops will still try to add necessary security group rules to the specified existing security group, despite that the name of field is "securityGroupOverride".*
```

Comment:
Thanks for the clarification! How about something more concise like:

```suggestion
- *Kops will add necessary rules to the security group specified in `securityGroupOverride`.*
```

Reply:
Sure. Done 👍